web/app/img/manifest.webmanifest

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "eLabFTW",
+  "icons": [
+    { "src": "app/img/icon-192.png", "type": "image/png", "sizes": "192x192" },
+    { "src": "app/img/icon-512.png", "type": "image/png", "sizes": "512x512" }
+  ]
+}
```

Comment:
be careful, this file is missing a newline at the end.

Reply:
I'll fix that. I also realized that I screwed up the path to `manifest.webmanifest`. Do you prefer to have inside `img` inside `app` or in the root?

Reply:
shouldn't it be at the web root? so in `web/` ?